pom.xml

```diff
@@ -331,12 +331,14 @@
             <goals>
               <goal>checkstyle</goal>
             </goals>
-            <phase>verify</phase>
+            <phase>validate</phase>
           </execution>
         </executions>
         <configuration>
           <linkXRef>false</linkXRef>
-          <failOnViolation>false</failOnViolation>
+          <violationSeverity>warning</violationSeverity>
```

Comment:
this _should_  be causing the build to fail at https://github.com/jenkinsci/prism-api-plugin/pull/41/commits/66dcb92a0ef09d7bb84de42c7f7c62cd965b0b2b but it doesn't seem to be 😢